### PR TITLE
Release v0.1.1-alpha.20

### PR DESCRIPTION
**Pull request by soapraj**
Making a new release as this fix #2097 was made after the last release.

I have made changes to lerna.json in the root of the repo and the one in the app-template within cli. Not sure if these changes are right, would love some more 👀 on this.

cc @spotify/backstage-core 

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "packages": ["packages/*", "plugins/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.1.1-alpha.18"
+  "version": "0.1.1-alpha.20"
```
Comment (soapraj) [inline lerna.json:5]: The lerna version doesn't seem to be updated when you do a yarn release. So after the previous alpha.19 release the version here was still pointing to alpha.18 and I think this is the version that gets picked up then you run create-app or create-plugin in the cli.

So updated this manually to match the new version.

File: packages/create-app/templates/default-app/lerna.json.hbs [hunk 2]
```diff
@@ -2,5 +2,5 @@
   "packages": ["packages/*", "plugins/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.1.0"
+  "version": "{{version}}"
```
Comment (soapraj) [inline packages/create-app/templates/default-app/lerna.json.hbs:5]: Converted this file to a template and changed the version to be populated by the CLI.

One of the reports in discord was that create-plugin didn't work and it picked up the version`0.1.0` for @backstage/cli and @backstge/theme from here. The create-plugin doesn't work as these versions don't exist and it be pointing to the latest version.

File: lerna.json (hunk 1 above)
Comment (freben) [reply lerna.json:5]: Oh interesting. That is indeed weird.

**Review by stefanalund — APPROVED** [review]

**Comment by freben** [comment]
SHIP? :)

**Pull request merged**